com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -219,6 +219,8 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
             }
 
             HLSLPROGRAM
+            #pragma exclude_renderers gles
```

Comment:
Should these actually be  #pragma exclude_renderers d3d11_9x gles gles3 glcore

Reply:
Yes indeed!
I will make this fix in the master branch if that's ok, because the correct fix may require to declare fallback shaders. Deferred render is actually not enabled in 9.x.x, so this PR is just fixing compiling errors.